### PR TITLE
fix(sandbox): ignore extra params on processor

### PR DESCRIPTION
**Pull request by roggervalf**

### CODE REVIEW

File: src/classes/child-processor.ts [hunk 1]
```diff
@@ -44,8 +43,15 @@ export class ChildProcessor {
       });
     }
 
+    const origProcessor = processor;
     if (processor.length > 1) {
```
Comment (roggervalf) [inline src/classes/child-processor.ts:47]: this path was not tested, when there are more than 1 parameter, we lose completed event

File: src/classes/child-processor.ts [hunk 2]
```diff
@@ -44,8 +43,15 @@ export class ChildProcessor {
...
-      processor = promisify(processor);
+      processor = function (arg: any) {
```
Comment (manast) [inline src/classes/child-processor.ts:48]: I am confused about this, if there are more than 1 argument, why do we only define 1 argument (arg)

Comment (manast) [reply src/classes/child-processor.ts:48]: In fact I do not understand why we need to separate between 1 and more than 1 🤔

**Review by manast — APPROVED** [review]
LGTM

**Pull request merged**